docs/source/notebooks/sampling_conjugate_step.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<p>I think @alexandorra removed <code>config InlineBackend.figure_format = 'retina'</code>  from the style guide in https://github.com/pymc-devs/pymc3/wiki/PyMC's-Jupyter-Notebook-Style/_compare/0f89ad07b00dd587686d783125c1cc2c04c01691...3a92c230b9dc8896d6c82a32f4ce25f8d9e73191</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4199/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>(only pointing this out in case you need to re-run the notebook anyway for some reason, else don't bother :) )</p>

Reply:
No problem, I reran it anyway to make sure it ran on my other computer.